src/store/initial-state.js

```diff
@@ -18,7 +18,7 @@ import {
  */
 export const createInitialState = () => ({
   chartSize: {},
-  flags: Flags.defaults(),
+  flags: { ...Flags.defaults(), expandAllPipelines: false },
```

Comment:
since it's no more a flag, do we need to add it in flag. Can it be it's own key like themes?

Reply:
I do agree its not a flag anymore (atleast not in settings) but we are updating the flags object when we click the expandAllPipelines icon. The issue happened as this property was missing in the flags object in initial state. If we want to move this out, that will be a refactor which includes the changes of the icon click as well.

I would rather have this inside an object. I was about to propose making the state object more structured :D .... like having all the nonPipelineSettings (theme, featureHintStep etc) into an object ?